libs/blocks/bulk/utils.js

```diff
@@ -0,0 +1,269 @@
+import { getImsToken } from '../../../tools/send-to-caas/send-utils.js';
+import { fetchWithTimeout } from '../caas/utils.js';
+import { loadScript } from '../../utils/utils.js';
+import {
+  getLocalStorage,
+  setLocalStorage,
+} from '../review/utils/localStorageUtils.js';
```

Comment:
Same as above, ideally shared utility functions should be located in a central location.

Reply:
Please see comment below.